packages/mux-player-react/src/index.tsx

```diff
@@ -180,6 +181,9 @@ const usePlayer = (
       return defaultHasChanged(playerEl, value, propName);
     }
   );
+  // NOTE: Somewhere in the codebase, `currentTime` is getting cast to a number, resulting in `NaN` + an error.
+  // This is a bandaid solution for now. (CJP)
+  useObjectPropEffect('currentTime', currentTime ?? 0, ref);
```

Comment:
is this internally in the components or in the examples?

is this for `NaN ?? 0` ?